File: Java/KodeProgram/Laptop.java
// Laptop.java
// Berisi definisi class Laptop, turunan dari Elektronik.
class Laptop extends Elektronik {
    private int ram;
    private int storage;
    private String processor;

    public Laptop(int id, String nama, double harga,
                  String brand, int garansi, int stok,
                  int ram, int storage, String processor) {
        super(id, nama, harga, brand, garansi, stok);
        this.ram = ram;
        this.storage = storage;
        this.processor = processor;
    }

    @Override
    public String getKategori() { return "Laptop"; }

    @Override
    public String getDetail() {
        return "RAM:" + ram + "GB, Storage:" + storage + "GB, " + processor;
    }
}

File: Python/KodeProgram/Laptop.py
# Laptop.py
# Turunan Level 3: Laptop
from Elektronik import Elektronik

class Laptop(Elektronik):
    def __init__(self, id, nama, harga, brand, garansi, stok, ram, storage, processor):
        super().__init__(id, nama, harga, brand, garansi, stok)
        self.__ram = ram
        self.__storage = storage
        self.__processor = processor

    def get_kategori(self):
        return "Laptop"

    def get_detail(self):
        return f"RAM:{self.__ram}GB, Storage:{self.__storage}GB, {self.__processor}"